Final Project/app.py

```diff
@@ -126,5 +126,11 @@ def logout():
     logout_user()
     return redirect(url_for("index"))
 
+@app.route("/post/create/<petId>", methods = ['POST'])
```

Comment:
May I suggest we just do @app.route("/posts", methods = ['POST']) and have the request body include the entire post rather than just the message. RESTful design indicates that method (in this case POST) should be used as the verb rather than including it in the route itself. REST is the industry standard:
https://restfulapi.net/rest-api-design-tutorial-with-example/.